task-sdk/tests/task_sdk/execution_time/test_supervisor.py

```diff
@@ -1409,6 +1415,9 @@ def test_handle_requests(
             3. Checks that the buffer is updated with the expected response.
             4. Verifies that the response is correctly decoded.
         """
+        mock_secrets_masker.return_value = SecretsMasker()
```

Comment:
Should we just mock the `SecretsMasker` in conftest. I had added the following which unblocked me in one of the PRs

https://github.com/apache/airflow/blob/main/task-sdk/tests/conftest.py#L131-L133

Pavan had attempted a fix in https://github.com/apache/airflow/pull/48901/

Reply:
Yeah i think we can do that, do you think we should do in a follow up to unblock this critical bug?

Reply:
Will take up later